test/config_test.py

```diff
@@ -0,0 +1,53 @@
+from contextlib import contextmanager
+import pytest
+
+from oci.oci import config
+
+
+@contextmanager
+def write_config_file(tmpdir, cfg_data):
+    cfg_file = tmpdir.join("oci.conf")
+    cfg_file.write(cfg_data)
+    try:
+        yield cfg_file
+    finally:
+        pass
```

Comment:
not needed, because tmpdir is a fixture that gives us a temporary directory.
The directory is removed after the test, so we don't need to cleanup.